FieldOpt/Utilities/math.hpp

```diff
@@ -51,7 +51,9 @@ inline T calc_median(std::vector<T> list) {
     assert(!list.empty());
     std::sort(list.begin(), list.end());
     size_t size = list.size();
-    if (size % 2 == 0)
+    if (size == 1)
```

Comment:
Regarding the description above: "If the values are integers and the list has an even number of elements, the result will be floored." Where are the elements being floored?